vision/classification/leafsnap30/README.md

```diff
@@ -14,19 +14,30 @@ LeafSnap30 is a Neural Network model trained on the [LeafSnap 30 dataset](https:
 Pytorch LeafSnap30 ==> ONNX LeafSnap30 
 
 ## Inference
-Step by step instructions on how to use the pretrained model and link to an example notebook/code. This section should ideally contain:
+The steps needed to run the pretrained model with the onnxruntime are implemented within the explainability library [dianna](https://github.com/dianna-ai/dianna) in this [code](https://github.com/dianna-ai/dianna/blob/main/dianna/utils/onnx_runner.py). An example [tutorial notebook](https://github.com/dianna-ai/dianna/blob/main/tutorials/lime_images.ipynb) shows on how to use the model with dianna.
 
 ### Input
-Input to network (Example: 224x224 pixels in RGB)
+The input to the model are 3-channel RGB images of shape ```3 x 128 x 128``` (colour channel first) as a numpy array of type ```float32```.
 
 ### Preprocessing
-Preprocessing required
+The input image is loaded to a nympy array. The pixel values are then converted to the 0-1 range. 
```

Comment:
Typo: nympy->numpy.
I would say "scaled" instead of "converted" in the second sentence.

Reply:
Indeed.